src/main/kotlin/com/shortener/exception/InvalidInputUrl.kt

```diff
@@ -0,0 +1,7 @@
+package com.shortener.exception
+
+class InvalidInputUrl : RuntimeException {
```

Comment:
Note: in kotlin everything is a RuntimeException. In other words, there are no "checked exceptions". 

So maybe it makes sense to extend `Exception` instead of `RuntimeException`

Reply:
Done, `Exception` it is